aws_s3_policies/aws_s3_bucket_lifecycle_configuration.py

```diff
@@ -1,15 +1,23 @@
+from panther_base_helpers import deep_get
+
 MAX_RETENTION_PERIOD = 365
 MIN_RETENTION_PERIOD = 90
 
 
 def policy(resource):
-    if resource["LifecycleRules"] is None:
+    if resource.get("LifecycleRules") is None:
         return False
 
-    for lifecycle_rule in resource["LifecycleRules"]:
-        if lifecycle_rule["Expiration"] is None or lifecycle_rule["Status"] != "Enabled":
+    for lifecycle_rule in resource.get("LifecycleRules", []):
+        if lifecycle_rule.get("Expiration") is None or lifecycle_rule.get("Status") != "Enabled":
```

Comment:
We can reduce this to just:
```suggestion
        if lifecycle_rule.get("Status") != "Enabled":
```
since the first check is just to make sure the check that used to be on line 12 is safe, which we don't need any more with `deep_get`.